great-idea-website/index.html

```diff
@@ -15,49 +15,87 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+    <header>
+    <div class="container">
+      <nav>
+    <a href="#"> Services </a>
+    <a href="#">Product</a>
+    <a href="#"> Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+        </nav>
```

Comment:
The indentation can be a bit better.  Proper indentation makes reading code a lot easier.  Pro-Tip.  If you right click on your html page inside VSC you should have an option to format code.  It's amazing